lib/index.js

```diff
@@ -42,7 +42,9 @@ $(document).ready(() => {
 
 const filterFoods = () => {
   var filter = $('#food-filter-input').val().toUpperCase()
-  var foodNameNodes = $('.food-item-name')
+
+  let foodNameNodes = findNodesForFilter()
```

Comment:
I like the functionality of reusing this method and passing in a variable that obtains the correct nodes depending on which page the user is on.